src/pages/Selector/container.jsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
+import { transformEmployees as transform } from './transform-employees';
 
-// temporarily disabling this rule until the function is fleshed out
-/* eslint-disable react/prefer-stateless-function */
 const SelectorContainer = Wrapped =>
-  class Selector extends React.Component {
+  class extends React.Component {
+    state = { employees: [] };
+
+    componentDidMount() {
+      this.fetchEmployees();
+    }
+
+    fetchEmployees = () => {
+      fetch('http://localhost:8081/api/users').then(resp => {
```

Comment:
Probably jumping the gun but something to think about when he gets down the line, obviously this url will change to be hosted somewhere. Maybe we should build out a baseConfig where it holds these urls or someway of switching out the urls based on the environment.